src/utils/push_array.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Does `array.push` for all `items`.
+ * Needed because `array.push(...items)` throws
+ * "Maximum call stack size exceeded" when items
+ * is too big of an array
```

Comment:
this seems like a pretty short line wrap length

Reply:
Adjusted 👍